lib/install/opsuser/opsuser_conf.go

```diff
@@ -39,11 +39,6 @@ var RoleDataCenter = types.AuthorizationRole{
 	Privilege: []string{
 		"Datastore.Config",
 		"Datastore.FileManagement",
-		"VirtualMachine.Config.AddNewDisk",
```

Comment:
These privileges should likely not be reduced for ESX. In that case, there should be two datacenter roles, one each for VC and ESX. Will confirm and update.

Reply:
@hickeng `vic-machine create --ops-grant-perms` doesn't work on the master branch either. We could revisit this piece too; I'll mention it in the follow-on issue for ESX support (#7588).